### ISSUE TITLE
abnormal fetal heart outflow tracts

### ISSUE THREAD

**Issue by sng2140** [obophenotype/human-phenotype-ontology#9583]
**Preferred term label:**
Abnormal outflow tracts

**Synonyms**

**Definition (free text, please give PubMed ID)**
Outflow tract abnormalities account for 20% of prenatally diagnosed CHD. Fetal ultrasound may not always be able to easily and accurately describe the outflow anomaly, but can detect abnormal anatomy. Often, follow up with a fetal echo is recommended, but fetal ultrasound is limited
https://www.iame.com/course/1100/take

**Parent term (use hpo.jax.org/app)**
Fetal anomaly--> abnormal fetal morphology-->abnormal fetal cardiovascular morphology

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@sng2140
added as HP:4000143  Abnormal fetal heart outflow tract
